devise_cas_authenticatable.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency(%q<devise>, [">= 1.2.0"])
   s.add_runtime_dependency(%q<rubycas-client>, [">= 2.2.1"])
+  s.add_runtime_dependency(%q<mongoid>, [">= 4.0.2"])
```

Comment:
Let's leave this out of the dependencies.  MongoDB should not be required in order to use devise_cas_authenticatable; if people want to use the new Mongo-based strategy, they'll already have mongoid in their Gemfile in all likelihood (and it will be easy for them to see what's wrong if they don't).